### PR TITLE
added_the_wikidocs_to_the_moja_globaldocs_repo

### PR DESCRIPTION
**Pull request by MercybirungiS**
# Description

### CODE REVIEW

**Comment by MercybirungiS** [comment]
I added the wiki docs to the moja global docs that are version controlled.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Hi @MercybirungiS 

The work is not completely correct. We would not like to add FLINT as a submodule but migrate the FLINT Wiki to the specific doc pages that I had shared with you.

You might want to convert those Markdown pages to RST and put them here.

**Comment by MercybirungiS** [comment]
okay let me work on that

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "FLINT"]
+	path = FLINT
+	url = https://github.com/moja-global/FLINT.git
```
Comment (HarshCasper) [inline .gitmodules:3]: Please remove this (delete the file)

File: docs/FLINT.wiki/1.-Moja-FLINT.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+**Welcome to the Full Land Integration Tool!**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.-Moja-FLINT.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/Home.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Welcome to the moja.global wiki!
```
Comment (HarshCasper) [inline docs/FLINT.wiki/Home.rst:1]: You can delete this

File: docs/FLINT.wiki/3.4-Module-Science.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
+**Empirical Growth Model**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/3.4-Module-Science.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/3.4-Module-Science.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+a larger proportion of tree growth will go into leaves than stems. The
+increase in carbon stock is associated with tree growth that is drawn
+from the atmosphere pool, ensuring mass balance.
+
+As with all models, the empirical growth model can be calibrated to suit
+the data that is available, and up-dated through time. However, this is
+also one of its limitations, in that it requires up-dating to respond to
+changes in climatic conditions; whereas hybrid models do not.
+
+**Hybrid forest growth**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/3.4-Module-Science.rst:29]: Use a proper heading here

File: docs/FLINT.wiki/3.4-Module-Science.rst [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+(2002) and Waterworth et al (2007).
+
+Input\|Output ------------------\|------------------------------ 1. Time
+step: monthly\|1. Age: Age 2. Region: Province\|2. Pool1: Stemwood 3.
+Species\|3. Pool2: Bark 4. Species: rootshoot\|4. Pool3: Leaves 5. Age:
+Age\|5. Pool4: Course roots 6. Pool1: Stemwood\|6. Pool5: Fine roots 7.
+Pool2: Bark\| 8. Pool3: Leaves\| 9. Pool4: Course roots\| 10. Pool5:
+Fine roots\| 11. Rainfall\| 12. Minimum Temperature\| 13. Maximum
+temperature\| 14. Vapour Pressure Deficit\| 15. Soil water holding
+capacity\|
```
Comment (HarshCasper) [inline docs/FLINT.wiki/3.4-Module-Science.rst:62]: The tables are not rendered correctly

File: docs/FLINT.wiki/3.4-Module-Science.rst [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+that is run. WOFOST allows users to model the potential production,
+attainable production, and actual production for a given site. WOFOST
+runs at daily time steps. More information on WOFOST is available at:
+www.wageningenur.nl/en/Expertise-Services/Research-Institutes/alterra/Facilities-Products/Software-and-models/WOFOST.htm
+
+Input\|Input ------------------\|------------------------------ 1. Time:
+Daily\|1. Age: Age 2. Crop Type\|2. Pool1: stem 3. Pool1: stem\|3.
+Pool2: Leaves 4. Pool2: Leaves\|4. Pool3: Storage organs 5. Pool3:
+Storage organs\|5. Pool4: Roots 6. Pool4: Roots\| 7. Maximum
+temperature\| 8. Minimum temperature\| 9. Global radiation\|
```
Comment (HarshCasper) [inline docs/FLINT.wiki/3.4-Module-Science.rst:79]: The tables are not rendered correctly

File: docs/FLINT.wiki/3.4-Module-Science.rst [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+manure \|
+| 4. Pool 3: Total organic carbon \|
+| 5. Pool 4: Microbial biomass carbon \|
+| 6. Rainfall \|
+| 7. Open pan evaporation \|
+| 8. Mean air temperature \|
+| 9. Clay content of soil \|
+| 10. Decomposability of incoming plant material \|
+| 11. Soil Cover \|
+| 12. Depth of soil layer sampled. \|
```
Comment (HarshCasper) [inline docs/FLINT.wiki/3.4-Module-Science.rst:105]: The tables are not rendered correctly

File: docs/FLINT.wiki/1.1-Pools.rst [hunk 9]
```diff
@@ -0,0 +1,32 @@
+**What is a pool?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.1-Pools.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/1.1-Pools.rst [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+A pool is a reservoir within which something can be stored and released.
+For example, a carbon pool is a reservoir from which carbon can be
+stored (sequestered and/or maintained) or emitted, such as debris.
+Within FLINT, each pool is attributed a value, for example tonnes
+carbon, and at each time step the FLINT can move stores from one pool to
+another using operations.
+
+**How are pools used in FLINT?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.1-Pools.rst:10]: Use a proper heading here

File: docs/FLINT.wiki/1.2-Operations.rst [hunk 11]
```diff
@@ -0,0 +1,48 @@
+**What is an operation?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.2-Operations.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/1.2-Operations.rst [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+An operation is a process within the FLINT that moves carbon stock
+between pools. Operations can reflect ongoing natural processes, such as
+growth, or specific events, natural or human induced, for example, a
+harvest moves plant material to products and debris pools, wildfire
+moves plant material to debris and atmosphere pools. The amount of stock
+moved during an operation is referred to as a ‘flux’. Operations allow
+the FLINT to track changes in carbon stock through time, including
+fluxes into and out of pools.
+
+**How are operations used in FLINT?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.2-Operations.rst:12]: Use a proper heading here

File: docs/FLINT.wiki/1.3-Module.rst [hunk 13]
```diff
@@ -0,0 +1,75 @@
+**What is a module?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.3-Module.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/1.3-Module.rst [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+
+A module is a self-contained set of operations that determine the state
+of, or change in, variables across a specified period of time for a
+single Simulation Unit in direct response to event notifications from
+the FLINT core system (Unit Controller). For example, the empirical
+forest growth module includes all the operations required to simulate
+biomass accumulation.
+
+**How do modules operate?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.3-Module.rst:10]: Use a proper heading here

File: docs/FLINT.wiki/1.3-Module.rst [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+Standard modules in FLINT
+
+The FLINT includes six standard modules for simulating changes in forest
+and crop systems, including changes in soil carbon. These modules each
+run for all simulations at a monthly step interval (see Temporal
+Distribution), and include: \* Empirical forest growth module \* Hybrid
+forest growth module \* WOFOST crop growth module \* Turnover module \*
+Decomposition module \* RothC soil carbon module
+
+**Modules for disturbance events**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.3-Module.rst:41]: Use a proper heading here

File: docs/FLINT.wiki/1.3-Module.rst [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+eligibility criteria (i.e. Simulation Unit characteristics that the
+event is allowed to affect). This information can be used to attribute
+the disturbance event to Simulation Units using a pre-determined
+criteria (e.g. randomly, weighted, or based on a unit characteristic).
+
+The follow is a disturbance events included in FLINT. Carbon pool events
+are those that directly affect one or more carbon pool, while non-carbon
+pool events indirectly affect one or more non-carbon pool.
+
+**Carbon pool events**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.3-Module.rst:63]: Use a proper heading here

File: docs/FLINT.wiki/1.4-Simulation-Units.rst [hunk 17]
```diff
@@ -0,0 +1,18 @@
+**What is a Simulation Unit?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.4-Simulation-Units.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/1.4-Simulation-Units.rst [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+
+A Simulation Unit is a unit for which a module is applied. A Simulation
+Unit can represent a spatial area, such as a pixel or forest stand, or
+it can represent an emissions source, such as livestock. Where the
+Simulation Unit refers to a geographically referenced area, it is known
+as a Land Unit.
+
+**How are Simulation Units used in the FLINT?**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/1.4-Simulation-Units.rst:9]: Use a proper heading here

File: docs/FLINT.wiki/2.-Data-Preparation.rst [hunk 19]
```diff
@@ -0,0 +1,127 @@
...
+Vectors are also a format of spatial files for GIS, however unlike
+rasters the information relates to points, lines or polygons, rather
+than pixels. For the purpose of the FLINT, as a minimum, vector files
+must…..[TBC]
+
+SQL Database (Relational) Structured Query Language (SQL) database is a
+relational database, meaning that the data is tabular format, where
+attributes are listed against a related to each record (row).
+
+**NoSQL Database**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/2.-Data-Preparation.rst:102]: Use a proper heading here

File: docs/FLINT.wiki/2.-Data-Preparation.rst [hunk 20]
```diff
@@ -0,0 +1,127 @@
...
+data-set is divided into smaller parcels. This is done to ensure that
+processing the raster can be carried out across multiple servers. For
+indexing, the raster of a country can be broken up into tiles, which can
+then be divided into blocks, which are divided into cells. For the
+FLINT, rasters are divided into 1 degree tiles, 0.1 degree blocks, and
+pixel resolution cells (depending on the data source). The 1 degree tile
+has been found to be the most efficient size, as they contain a large
+amount of data, without compromising the processing capacity.
+
+**Vector**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/2.-Data-Preparation.rst:91]: Use a proper heading here

File: docs/FLINT.wiki/2.-Data-Preparation.rst [hunk 21]
```diff
@@ -0,0 +1,127 @@
...
+-  for example, ask for soil ID, get all soil parameters.
+-  Table
+-  A table is effectively a database that can be looked up. For example,
+   in the CBM, get table functions are used for:
+
+   -  hardwood volume to biomass parameter
+   -  softwood merchantable volume
+   -  hardwood merchantable volume
+
+**Data Provider types**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/2.-Data-Preparation.rst:64]: Use a proper heading here

File: docs/FLINT.wiki/2.-Data-Preparation.rst [hunk 22]
```diff
@@ -0,0 +1,127 @@
+**Input data from the FLINT**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/2.-Data-Preparation.rst:1]: Use a proper heading here

File: docs/FLINT.wiki/2.-Data-Preparation.rst [hunk 23]
```diff
@@ -0,0 +1,127 @@
...
+Data Providers (in much the same way Modules can be provided in external
+libraries) as long as they match 1 of the defined external data
+categories (Raster, Vector, etc.). Through this functionality, modules
+can remain independent from the core of the FLINT, significantly
+increasing the flexibility of the FLINT. This also allows the modules to
+easily use what is available, and run spatially without reference to a
+specific location or time. This will allow the FLINT to run spatially
+without having to set individual modules up to do so.
+
+**Types of data**
```
Comment (HarshCasper) [inline docs/FLINT.wiki/2.-Data-Preparation.rst:21]: Use a proper heading here

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Review Round-1

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The docs are not referenced anywhere across the website. I will close this PR in favor of a different one.

**Pull request closed without merging**